more_itertools/more.py

```diff
@@ -1222,24 +1222,23 @@ def numeric_range(*args):
     """An extension of the built-in ``range()`` function whose arguments can
     be any orderable numeric type.
 
-    If the *start* argument is omitted, it defaults to ``0``. If the *step*
-    argument is omitted, it defaults to ``1``. If *step* is zero,
-    ``ValueError`` is rasied.
-
-    With only *stop* specified:
+    With only *stop* is specified, *start* defaults to ``0`` and *step*
```

Comment:
The _is_ is unnecessary. This PR is teaching me I should just listen to you the first time.